requirements.in

```diff
@@ -47,3 +47,9 @@ notifications-python-client
 #   monitoring and observability
 #-----------------------------------
 sentry-sdk
+
+
+#-----------------------------------
+#   Background Tasks
+#-----------------------------------
+celery==5.4.0
```

Comment:
Can we use `celery[redis]` instead? (this should scope down the included dependencies)

Reply:
Changed it to celery[redis].